todo_app/data/view_model.py

```diff
@@ -0,0 +1,33 @@
+from todo_app.data.item import Item
+
+class ViewModel:
+    def __init__(self, items: list[Item]):
+        self._items = items
+ 
+    @property
+    def todo_items(self) -> list[Item]:
+        output = []
+        
+        for item in self._items:
+            if item.status == "To Do":
+                output.append(item)
+        
+        return output
```

Comment:
Nice 👍 Not a recommendation, but I always think interesting to see the list comprehension approach as a comparison - arguably harder to read, but much more concise:

```python
    @property
    def todo_items(self) -> list[Item]:
        return [item for item in self._items if item.status == "To Do"]
```